pom.xml

```
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.57</version>
+        <version>4.74</version>
```

Most recent release of plugin parent pom
```suggestion
        <version>4.76</version>
```